Blocks World for Teams.md

```diff
@@ -0,0 +1,75 @@
+# Summary
+This studied examined the communication among teams by having teams of four people search for and deliver a specified series of colored blocks as quickly as possible.
+
+>Low-complexity task
+>> Participants were required to collect six blue-colored blocks
+>
+>Medium-complexity task
+>> Participants were required to search for and deliver 16 blocks, which were divided into two sequences that could be collected in parallel. One sequence called for pink- and green-colored blocks in alternating order, while the other required white and blue blocks in alternating order. Each of the four team members was assigned a specific color.
+>> 
+>> In one of the two medium-complexity tasks, it incorporated a “communication failure” in which two of the four agents were unable to communicate with each other for a prespecified duration within the task.
+>>
+>High-complexity task
+>> Participants were required to collect six blocks in a random sequence of colors that was predetermined during the experiment design phase
+
+# References
+Main paper: https://journals-sagepub-com.proxy.library.upenn.edu/doi/10.1177/0018720816639712
+Supplementary: file:///Users/katelynbottcher/Downloads/961321058-MIT.pdf
+
```

Comment:
Here's the GitHub page to download and run the game: https://github.com/eishub/BW4T/tree/master/doc/src/main